Makefile

```diff
@@ -461,7 +461,7 @@ _up:
 		--file docker-compose.yml \
 		up --force-recreate --always-recreate-deps --renew-anon-volumes \
 		--exit-code-from pulumi \
-		pulumi
+		pulumi && echo "✅ Pulumi deployed successfully ✅"
```

Comment:
I added this because I've been constantly annoyed by the big scary red text "ERROR EXIT CODE 0"  here 
![image](https://user-images.githubusercontent.com/69007229/172226492-0df19c27-7c17-47b6-ab97-3ece739534ba.png)